pandas/tests/arithmetic/conftest.py

```diff
@@ -40,8 +57,19 @@ def one(request):
 
 @pytest.fixture(params=zeros)
 def zero(request):
-    # For testing division by (or of) zero for Index with length 5, this
-    # gives several scalar-zeros and length-5 vector-zeros
+    """
+    Several types of scalar zeros and length 5 vectors of zeros. For
+    testing division by (or of) zero.
+
+    Uses vector of length 5 for broadcasting with `numeric_idx` fixture,
```

Comment:
Can you model the extended description here after what you have in the other method? I find that quite nice, but don't really understand this one